### DISCUSSION TITLE
Compatibility with EZ1-M (built in ECU and new app Easy Power)

### DISCUSSION THREAD

**Discussion by andre68723** [ksheumaker/homeassistant-apsystems_ecur#197] [Q&A]
Hello,
is this plugin compatible with the EZ1-M microinverter?
The successor of DS3* has built-in wifi and comes with a new app "AP easy power"

Ref https://emea.apsystems.com/diy/

Thanks

**Comment by HAEdwin** [comment]
The EZ1 eliminates the need for an ECU (Energy Communication Unit) and therefore uses a different approach to installation and monitoring. So the answer is no unfortunately.

**Reply by andre68723** [reply]
which means currently no possibility to integrate into Home Assistant, correct?

**Reply by HAEdwin** [reply]
Not with this integration, there might be another solution available to integrate the inverter with HA by using scraping or something similar.

**Comment by HAEdwin** [comment]
I do not have an inverter from the EZ series to see if it can be added to this integration. Testing the integration therefore becomes more difficult.
